src/components/projects.module.css

```diff
@@ -122,6 +122,7 @@
   margin-left: .5vw;
   margin-right: 2vw;
   font-size: 2vw;
+  margin-bottom: -.75vw;
```

Comment:
Why is vw (vertical width) being used for a margin bottom versus vh (vertical height)?

Reply:
Yeah, I probably went a little nuts with the vw. I am just trying to figure out the responsive design structure. I think you're totally right about rem and em as the designer of the site I am doing freelance said he never heard of a vw. As for the using vw or vh, I was just being consistent and thinking more of it as a measurement over anything else